R/toolSplitBiomass.R

```diff
@@ -0,0 +1,38 @@
+#' Split Biomass into modern and traditional
+#'
+#' We assume that below a given GDP/cap level, all biomass is traditional and
+#' above a higher limit, all biomass is modern with a linear transition between
+#' the limits.
+#'
+#' @param x MagPIE object including biomass data
+#' @param gdppop MagPIE object with GDP/cap data
+#' @param split character, name of item to split
+#' @param into character vector of length two with the names of the split items
+#' @param limits numeric vector of length two with the corresponding GDP/cap
+#'   limits. The default values used to be 10k and 15k USD/cap converted from
+#'   2005 to 2017 dollars.
+#' @returns MagPIE object including split biomass data
+#'
+#' @author Robin Hasse
+#'
+#' @importFrom dplyr %>%
+#' @importFrom magclass time_interpolate setNames getYears
+#' @export
+
+toolSplitBiomass <- function(x,
+                             gdppop,
+                             split = "biomass",
```

Comment:
I was a bit irritated that the function is called ```toolSplitBiomass```, but actually would allow to split something else. In general, I think this is nevertheless a good solution - the only disadvantage I see is that if anyone ever wants to split something other than biomass they might not realize they can use this function.

Reply:
Thanks for flagging this. I agree and would still go with the current name as I think reusing this is unlikely and the name is the clearest for it's current purpose.